google/cloud/bigtable/instance.py

```diff
@@ -644,6 +644,184 @@ def list_tables(self):
 
         return result
 
+    def backup(self, backup_id, cluster_id=None, table=None, expire_time=None):
+        """ Factory to create a Backup within this Instance.
+
+        :type backup_id: str
+        :param backup_id: The ID of the Backup to be created.
+
+        :type cluster_id: str
+        :param cluster_id: (Optional) The ID of the Cluster. Required for
+                           calling 'delete', 'exists' etc. methods.
+
+        :type table: :class:`~google.cloud.bigtable.table.Table`
+        :param table: (Optional) The Table create the Backup from.
+            Required, if the `create` method needs to be called.
+
+        :type expire_time: :class:`datetime.datetime`
+        :param expire_time: (Optional) The expiration time of this new Backup.
+            Required, if the `create` method needs to be called.
+        """
+        try:
+            return Backup(
+                backup_id,
+                self,
+                cluster_id=cluster_id,
+                table_id=table.name.split("/")[-1],
+                expire_time=expire_time,
+            )
+        except AttributeError:
+            return Backup(
+                backup_id,
+                self,
+                cluster_id=cluster_id,
+                table_id=table,
+                expire_time=expire_time,
+            )
```

Comment:
I think it can be done without `try-except`. `table.name` seems to be returning `str` always, so we can just write:
```python
table_id = table.name.split("/")[-1]
```
I've tried strings with slashes and without - `split()` always returns a `list`, even if there was no separator symbol in the string. See:

![Безымянный](https://user-images.githubusercontent.com/26030071/85004178-eded9000-b15f-11ea-8dba-1ad0fae40aad.png)